TF_Settings_and_Tooling_Unity/Assets/TouchFree/SettingsUI/Scripts/Configuration/Screens/AdvancedSettingsScreen.cs

```diff
@@ -89,6 +91,24 @@ public void SetFileLocation()
             }
         }
 
+        public void OpenTouchFreeLogFileLocation()
+        {
+            OpenFolderInFileExplorer(fileLocation.text);
```

Comment:
This path isn't for the log, this is for the config files, the TouchFree Service log is currently at C:\Program Files\Ultraleap\TouchFree\Wrapper\log.txt due to the use of the wrapper